tests/fixtures.py

```diff
@@ -847,20 +892,65 @@ def dataset_config_preview(
     dataset_config.delete(db)
 
 
+def load_dataset(filename: str) -> Dict:
+    yaml_file = load_file(filename)
+    with open(yaml_file, "r") as file:
+        return yaml.safe_load(file).get("dataset", [])
+
+
 @pytest.fixture
 def example_datasets() -> List[Dict]:
     example_datasets = []
     example_filenames = [
         "data/dataset/postgres_example_test_dataset.yml",
         "data/dataset/mongo_example_test_dataset.yml",
+        "data/dataset/snowflake_example_test_dataset.yml",
```

Comment:
My plan from here out is to avoid adding any subsequent datasets to this fixture, preferring to have them loaded in by a dedicated fixture for each one.